packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -43,9 +45,15 @@ class TabBarTheme with Diagnosticable {
   /// Overrides the default value for [TabBar.indicator].
   final Decoration? indicator;
 
+  /// Default value for [TabBar.indicatorColor].
```

Comment:
Just to be consistent with the other property descriptions: 

```suggestion
  /// Overrides the default value for [TabBar.indicatorColor].
```